lib/db/schema/patch-031-032.sql

```diff
@@ -0,0 +1,94 @@
+CREATE TABLE IF NOT EXISTS securityEventNames (
+  id INT AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  UNIQUE INDEX securityEventNamesUnique(name)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS securityEvents (
+  uid BINARY(16) NOT NULL,
+  nameId INT NOT NULL,
+  FOREIGN KEY (nameId) REFERENCES securityEventNames(id) ON DELETE CASCADE,
+  tokenId BINARY(32),
+  verified BOOLEAN,
+  ipAddr BINARY(32) NOT NULL,
```

Comment:
It may be worth explicitly naming this column `ipAddrHmac` or similar so that we don't confuse ourselves in future, especially since it's binary and hence could be easy for a future reader to re-interpret as an actual ip address.